gp-templates/helper-functions.php

```diff
@@ -108,6 +108,10 @@ function gp_sort_glossary_entries_terms( $glossary_entries ) {
 			$terms[] = substr( $term, 0, -1 ) . 'ves';
 		} elseif ( 'fe' === substr( $term, -2 ) ) {
 			$terms[] = substr( $term, 0, -2 ) . 'ves';
+		} elseif ( 'e' === substr( $term, -1 ) ) {
+			$terms[] = substr( $term, 0, -1 ) . 'ed';
+			$terms[] = substr( $term, 0, -1 ) . 'ion';
```

Comment:
I think adding `ion` should only be added for specific last characters (after removing `e`).  Some examples where `ion` doesn't make sense: `wave`, `house`, `care`, `sensitive`

Reply:
Could it be the case only for words ending with "te"?

Reply:
Here is a comprehensive list of suffix rules for -ion
https://www.thefreedictionary.com/Commonly-Confused-Suffixes-tion-vs-sion.htm

There are many other cases, I'll add this to the issue so it can be discussed also.